packages/task/src/browser/task-action.ts

```diff
@@ -0,0 +1,52 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable, inject } from 'inversify';
+import { TaskService } from './task-service';
+import { QuickOpenBaseAction, QuickOpenActionProvider, QuickOpenAction } from '@theia/core/lib/browser/quick-open/quick-open-action';
+
+@injectable()
+export class ConfigureTaskAction extends QuickOpenBaseAction {
```

Comment:
please keep file named after main class, i.e. `task-action-provider.ts`, it is a part of naming conventions for Theia to simplify searching classes by file names

Reply:
`task-action.ts` ---> `task-action-provider.ts`
`quick-open-action.ts` ---> `quick-open-action-provider.ts`